### PR TITLE
CKEditor in admin inline bug fix

### PR DESCRIPTION
**Pull request by evgkirov**
There is a bug which prevents editors in admin inlines from being properly initialized.

For example if you have some admin.StackedInline with extra = 0 and RichTextField inside it, the field appears to be unclickable after you hit "Add a row". So you need a lazy initialization for these.

So I cherry-picked a solution from another fork that was outdated.

### CODE REVIEW

File: ckeditor/templates/ckeditor/widget.html [hunk 1]
```diff
@@ -1,10 +1,3 @@
...
-<script type="text/javascript">
-    if (typeof CKEDITOR == "undefined") {
-        $( document ).ready(function() {
-            CKEDITOR.replace("{{ id }}", {{ config|safe }});
-        });
-    } else {
-        CKEDITOR.replace("{{ id }}", {{ config|safe }});
-    }
-</script>
+<div style="margin-left: 106px">
```
Comment (evgkirov) [inline ckeditor/templates/ckeditor/widget.html:1]: This is a dirty hack for correct positioning the editor in admin. I hope there is another way to work this around.
536104bf0077913664b6777e

**Comment by riklaunim** [comment]
Maybe some option to pass you own template or JS/Styles. Will have to think about it.

**Comment by riklaunim** [comment]
Can you provide admin.py and models.py example to reproduce this? (and django version; can't get it on 1.6).

**Comment by evgkirov** [comment]
models.py

```
from django.db import models
from django.utils.translation import ugettext_lazy as _
from ckeditor.fields import RichTextField


class Entry(models.Model):
    title = models.CharField(verbose_name=_('title'), max_length=255)
    text = RichTextField(verbose_name=_('text'), blank=True)

    def __unicode__(self):
        return self.title

    class Meta:
        verbose_name = _('blog entry')
        verbose_name_plural = _('blog entries')

class QA(models.Model):

    entry = models.ForeignKey(Entry)
    question = RichTextField(verbose_name=_('question'), config_name='question')
    answer = RichTextField(verbose_name=_('answer'),  config_name='answer')

    class Meta:
        verbose_name = _('question')
        verbose_name_plural = _('questions and answers (for interviews)')
```

admin.py

```
from django.contrib import admin
from django.db import models
from django.utils.translation import ugettext_lazy as _
from modeltranslation.admin import TabbedTranslationAdmin, TranslationStackedInline

from .models import Entry, QA


class QAInline(TranslationStackedInline):
    model = QA
    extra = 0
    fields = ('question', 'answer')


class EntryAdmin(TabbedTranslationAdmin):
    list_display = ('title',)
    list_display_links = list_display[:2]
    fields = ('title', 'text')
    inlines = (QAInline, )

admin.site.register(Entry, EntryAdmin)
```

Django 1.6.4

**Comment by riklaunim** [comment]
Looks good but that jQuery will cause problems to some users which will have already loaded jQuery. And it probably should use //url and not http://url to avoid problems when running site with https://

Will check and tune it.

**Pull request merged**